### PR TITLE
Instructions for database init

### PR DESCRIPTION
**Pull request by falapen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,29 @@ chmod +x hook_setup.sh
 in order to set up git hooks
 
 ssh-working hell yeahhhh!!
```
Comment (kramstrom) [inline README.md]: remove

**Review by kramstrom — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -16,3 +16,29 @@ chmod +x hook_setup.sh
...
+
+## Database:
+
+To correctly setup a database, you need to run a replica set and be connected to that. This can be done locally for development purposes using the run-rs npm package. To start it (given that mongodb service 4.0 or above is installed) run the following command (remember the datafolder is always relative the path you run from).
+```
+run-rs --keep --mongod --dbpath <path/to/data/folder> &
+```
+start the mongo-shell and make sure you use the database 'cloudnews' using
```
Comment (robertcarlsson) [inline README.md]: One important thing is to be on the primary node, or does these commands work on secondary?

**Review by robertcarlsson — CHANGES_REQUESTED** [review]
fix this

**Comment by fsandelin** [comment]
Can you fix the requested changes or do you need help (refer to @antoncarlsson in that case)?

File: README.md (hunk 2 above)
Comment (falapen) [reply README.md]: k

**Comment by kramstrom** [comment]
Honestly @kramstrom  what is taking so long

File: README.md (hunk 1 above)
Comment (falapen) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (falapen) [reply README.md]: Done

**Review by robertcarlsson — APPROVED** [review]
Very good

**Review by kramstrom — APPROVED** [review]

**Comment by kramstrom** [comment]
Took you long enough

**Pull request merged**